cache/slru_cache.cpp

```diff
@@ -292,6 +295,21 @@ int slru_cache_t::remove(const unsigned char *id, uint64_t ioflags) {
 	TIMER_STOP("remove.find");
 
 	if (it) {
+		if ((cmd->cmd == DNET_CMD_DEL_NEW) && (request.ioflags & DNET_IO_FLAGS_CAS_TIMESTAMP)) {
+			auto cache_ts = it->timestamp();
+
+			// cache timestamp is greater than timestamp of the data to be removed
+			// do not allow it
+			if (dnet_time_cmp(&cache_ts, &request.timestamp) > 0) {
+				const std::string cache_ts_string = dnet_print_time(&cache_ts);
+				const std::string request_ts_string = dnet_print_time(&request.timestamp);
+				DNET_LOG_ERROR(m_node, "{}: cas: cache data timestamp is larger than data to be "
```

Comment:
Variant from `eblob_backend.cpp` is more suitable: `data timestamp is higher than request timestamp`.